conf/modules.config

```diff
@@ -236,7 +236,7 @@ process {
     // SUBWORKFLOW: READ_COVERAGE
     //
     withName: ".*:.*:READ_COVERAGE:MINIMAP2_ALIGN" {
-        ext.args = {"-ax ${meta.readtype.equals("hifi") ? "map-hifi" : meta.readtype.equals("clr") ? "map-pb" : meta.readtype.equals("ont") ? "map-ont" : meta.readtype.equals("illumina") ? "sr" : ""} --cs=short ${reference.size() > 2.5e9 ? (" -I" + Math.ceil(reference.size()/1e9)+"G") : ""}" }
+        ext.args = {"-ax ${meta.readtype.equals("hifi") ? "map-hifi" : meta.readtype.equals("clr") ? "map-pb" : meta.readtype.equals("ont") ? "map-ont" : meta.readtype.equals("illumina") ? "sr" : ""} ${reference.size() > 2.5e9 ? (" -I" + Math.ceil(reference.size()/1e9)+"G") : ""}" }
```

Comment:
Change `-ax` to `-x` here as `-a` gets added by the `bam_output` option anyway (and this would force BAM output even if PAF was indicated by the other module options.

Reply:
that was from my copy and paste, resolved .